php/class-plugin.php

```diff
@@ -36,7 +36,6 @@ public function __construct() {
 	 */
 	public function init() {
 		$this->config = apply_filters( 'customize_pane_resizer_plugin_config', $this->config, $this );
-		$this->min = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
```

Comment:
Sure, that's fine. Removing the minification makes sense. But please do fix
the symlinks to replace them with the regular file equivalents.